### PR TITLE
Memory Fix, Volumetric Limiter

### PR DESCRIPTION
**Pull request by owenmylotte**
Reduces the memory consumption of the radiation solver and protects the volumetric radiation from eos integration failures. Additional changes to enforce positivity in the absorption model.

### CODE REVIEW

File: src/eos/radiationProperties/zimmer.cpp [hunk 1]
```diff
@@ -64,7 +64,11 @@ PetscErrorCode ablate::eos::radiationProperties::Zimmer::ZimmerFunction(const Pe
         pCO = (density * UGC * YinCO * temperature) / (MWCO * 101325.);
 
         /** The resulting absorptivity is an average of species absorptivity weighted by partial pressure. */
-        *kappa = pCO2 * kappaCO2 + pH2O * kappaH2O + pCH4 * kappaCH4 + pCO * kappaCO;
+        *kappa = 0;
```
Comment (owenmylotte) [inline src/eos/radiationProperties/zimmer.cpp:67]: This enforces positive values out of the absorption model.

File: src/radiation/radiation.cpp [hunk 2]
```diff
@@ -738,7 +724,6 @@ void ablate::radiation::Radiation::EvaluateGains(Vec solVec, ablate::domain::Fie
     VecRestoreArrayRead(solVec, &solArray);
     VecRestoreArrayRead(auxVec, &auxArray);
     VecRestoreArrayRead(faceGeomVec, &faceGeomArray) >> checkError;
-    EndEvent();
```
Comment (mmcgurn) [inline src/radiation/radiation.cpp:741]: It looks like after the rebase we may be missing some Start/End events

**Review by mmcgurn — CHANGES_REQUESTED** [review]

File: src/radiation/radiation.cpp (hunk 2 above)
Comment (mmcgurn) [reply src/radiation/radiation.cpp]: I added back the events and pushed to the PR branch.

**Comment by owenmylotte** [comment]
Thank you for fixing this!

**Comment by mmcgurn** [comment]
This will be replaced with an updated PR using PetscSF for rad.

**Pull request closed without merging**